.generator/templates/model_utils.mustache

```diff
@@ -1368,10 +1369,6 @@ def get_oneof_instance(cls, model_kwargs, constant_kwargs, model_arg=None):
     if len(cls._composed_schemas['oneOf']) == 0:
         return None
 
-    # Buld a configuration that doesn't discard unknown keys for primitive types
-    discard_configuration = copy.deepcopy(constant_kwargs["_configuration"])
-    discard_configuration.discard_unknown_keys = False
```

Comment:
this is removed on purpose ?

Reply:
Yep, the `must_convert` change covers that area now.